src/NodeDev.Blazor/Index.razor

```diff
@@ -104,14 +98,16 @@
 
     protected override void OnInitialized()
     {
-        if (File.Exists("project.json"))
```

Comment:
We should probably find a place to auto-load the project like it used to. Since from what I understand the new behavior will force the user to re-upload the project each time

Reply:
Agreed. It is not really useful and we don't have access to the path to save the project either.

Reply:
@JordanFiset With the latest changes does it auto load the last project ?

Reply:
No but if you click open, the project should be in the list if you haven't changed the project directory. I can add that feature.